examples/typescript-kubernetes/.gitignore

```diff
@@ -0,0 +1,7 @@
+*.d.ts
+*.js
+node_modules
+cdktf.out
+terraform.tfstate*
+.gen
+.terraform
```

Comment:
We need to add `!tsconfig.json` plus the actual file here. I'll create a PR to add this to the template